docs/patterns/heading-icon.md

```diff
@@ -9,7 +9,7 @@ layout: default
 
 A header can be emphasised by adding an icon alongside the text.
 
-<a href="/examples/patterns/heading-icon/"
+<a href="/examples/patterns/heading-icon/heading-icon"
```

Comment:
Either works but I'll add one for consistency, I guess

Reply:
Ah, before one wouldn't work

Reply:
It works, but it is definitely worse without the `/` because `/examples/patterns/heading-icon/heading-icon` gets redirected to `/examples/patterns/heading-icon/heading-icon/`. One extra round-trip for the user, one extra request per visitor on our site, one extra line in our logs etc.

In every case, if a URL redirects to another URL, we should always link to the target URL unless we have a good reason.